### ISSUE TITLE
Field Lenght

### ISSUE THREAD

**Issue by GustavoSjcBr** [asikart/userxtd#41]
Hi All,

userXTD is a great job. Congratulations.

In field ABOUT, textarea, we are limited to 255 characters lenght.

I changed length of field "value" to 4096 in table "userxtd_profiles" and the behaviour was great.

Is it a good idea to new versions ?

Regards,

Gustavo.
Mysql  5.6.24 - Community Server 
Apache 2.4.12
Joomla 3.4.4

**Comment by asika32764** [comment]
Yes it is a good idea to change the field length to make profile more flexible,

I will add this in future version.
